Server/Item.cs

```diff
@@ -1089,8 +1074,17 @@ private void VerifyCompactInfo()
 				return;
 			}
 
-			var isValid = info.m_Name != null || info.m_Items != null || info.m_Bounce != null || info.m_HeldBy != null || info.m_BlessedFor != null ||
-						  info.m_Spawner != null || info.m_TempFlags != 0 || info.m_SavedFlags != 0 || info.m_Weight != -1;
+			var isValid = info.m_Name != null 
+					   || info.m_Items != null 
+					   || info.m_Bounce != null 
+					   || info.m_HeldBy != null 
+					   || info.m_BlessedFor != null 
+					   || info.m_Spawner != null 
+					   || info.m_TempFlags != 0 
+					   || info.m_SavedFlags != 0 
+					   || info.m_Weight != -1 
+					   || info.m_VisibilityList != null
+					   ;

```

Comment:
Weird spacing format?

Reply:
On purpose.... making things easier for others to merge in future.